Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nvidia/cuda:11.1-cudnn8-devel-ubuntu18.04
+FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04
```

Comment:
This doesn't solves #112

Reply:
It does. I create a docker image and it can successfully train a model on [kubeflow](https://datasetinsights.endpoints.unity-ai-thea-test.cloud.goog/_/pipeline/#/runs/details/8a4020f4-92aa-4730-846b-0289dc42c3da).

Reply:
Using SHA256 worked for me instead of tags.